_pages/community.html

```diff
@@ -77,9 +77,7 @@ <h4 class="card-title">Better Informatics</h4>
                 <ul class="list-group twitter-links">
                     <!-- Header -->
                     <li class="list-group-item header">
-                        <div class="d-flex w-100">
-                            <h4 class="mb-1">Follow us on Twitter</h4>
-                        </div>
+                        <h5 style="text-align: center; width: 100%; margin-bottom: 0">Follow us on Twitter</h5>
```

Comment:
this should work (more concise) `style: "margin: 0 auto"`

- `0` is for top and bottom (i guess)
- `auto` is for left and right (i guess), and automagically centers (works brilliantly for images)